### DISCUSSION TITLE
BpyTOP Port

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2006] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the suggested port be themed?

I created themes for Latte, Frappe, Macchiato, and Mocha for BpyTOP. 

I feel like it's pretty much done, though I will still probably tweak it over time. 

### Any additional comments?

I have completed the port and I am looking for approval or review before it gets approve.

Thank you!

URL of port: https://github.com/Arkwin/catppuccin-bpytop

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by unseen-ninja** [comment]
Hey there! Thanks for your work. 
We actually have an existing port for [BTOP](https://github.com/catppuccin/btop) which also works with BPyTOP as well. Instead of making a whole new Repository I'd opt to unify them in one repository.
Since the BTOP port has been untouched for quite a while it would probably best to check if your port is also backwards compatible and just take that – as judging by the screenshots they do look a little more refined. c:
If they are both interchangable for BTOP and BPyTOP it wouldn't even introduce any more maintenance.
Would you be fine with that idea?

**Reply by ghost** [reply]
Hey!

My mistake! When I did my search I did bpytop and htop for repos! I should have just searched top! I do not know about more refined, their repo looks pretty good. I just sort of threw mine together in the last 2 days. 

I am fine with however you want to do them. I like some aspects of their version like the blue processes. Do you want me to create a pull request on the BTOP repo with updates? 

Also, installed BTOP. They are backwards compatible.

**Reply by unseen-ninja** [reply]
Oh, yeah.. the screenshots also show slitghtly different things that's why they look different in certain areas. But I still think that some of your colour choises are better suited – especially looking at the graphs. If you could make a PR to the dev branch on the BTOP repo that would be lovely!

Also no worries.. Sometimes it just happens that we don't see things. c:

**Reply by ghost** [reply]
Sounds good. 

Should I close this discussion?

**Reply by unseen-ninja** [reply]
I'd leave it open for now and close it later once everything is merged together. That way we can reference the push over here in case someone stumbled across it in the future.

**Reply by ghost** [reply]
Sounds good. I will work on creating a PR to the dev branch. :)

**Comment by ghost** [comment]
PR to the dev branch submitted. :)

**Reply by ghost** [reply]
removed my catppuccin-bpytop repo, 

Forked btop, made all of my edits there, created a PR.

https://github.com/Arkwin/btop

**Comment by unseen-ninja** [comment]
Closed as per [#b5004e2](https://github.com/catppuccin/btop/commit/b5004e229e9a444f25e2ec126d74fe1527d7ec01) on the [Btop Repo](https://github.com/catppuccin/btop).
